deploy_from_github_zip.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ $# -eq 0 -o $# -gt 3 ] && {
+    echo "Usage: deploy_from_github_zip.bash <new project name> [GitHub-user/repository-name] [branch]"
+    exit 1
+}
+
+project=$1
+repo=${2-LSSTDESC/start_paper}
+branch=${3-master}
```

Comment:
Also its a bit odd that the folder that I create contains another copy of the script - another reason to use different branches for documentation and scripts, compared to clean template files!